eng/_util/cmd/run-builder/systemfips_fallback.go

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//go:build !windows
+// +build !windows
+
+package main
+
+// enableSystemWideFIPS is a no-op because the current platform either doesn't support or doesn't
+// require system-wide FIPS to be enabled to run tests.
+func enableSystemWideFIPS() (restore func(), err error) {
+	return nil, nil
```

Comment:
Do we want to log here? Just so we know that the fallback is being used and in case for some reason it gets called on windows?

Reply:
Wouldn't hurt! I think lack of logs is technically enough as far as diagnosis goes (it will always log *something* in the other case), but I do think it'd be worth it to call it out in case it catches the eye.